dev/Repeater/TestUI/Samples/FlexboxGrowPage.xaml

```diff
@@ -0,0 +1,126 @@
+﻿<Page
+    x:Class="MUXControlsTestApp.Samples.FlexboxGrowPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp.Samples"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:Microsoft.UI.Xaml.Controls"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+    <StackPanel>
+        <StackPanel.Resources>
+            <Style TargetType="TextBlock">
+                <Setter Property="Foreground" Value="White"/>
+                <Setter Property="VerticalAlignment" Value="Center"/>
+                <Setter Property="HorizontalAlignment" Value="Center"/>
+                <Setter Property="FontWeight" Value="Bold"/>
+                <Setter Property="FontSize" Value="24"/>
+            </Style>
+            <Style TargetType="Grid">
+                <Setter Property="Background" Value="Orange"/>
+             <!--   <Setter Property="MinWidth" Value="80"/> -->
```

Comment:
I see that this line was commented out from the previous version of these test files. And replaced with `Basis`. What was the behavior you were seeing with `MinWidth` that caused you to remove it? I think there may be something interesting to this, as `MinWidth` is a XAML intrinsic layout value, and this likely fits into the "How do XAML layout and Flexbox layout combine when there are overlapping concepts?"